secator/runners/_helpers.py

```
@@ -99,15 +98,19 @@
 		ctx = {}
 	all_results = []
 	errors = []
+	key = ctx.get('key', 'unknown')
 	if not isinstance(extractors, list):
 		extractors = [extractors]
 	for extractor in extractors:
 		try:
 			extractor_results = process_extractor(results, extractor, ctx=ctx)
+			debug(f'extracted {len(extractor_results)} / {len(results)} for key "{key}" with extractor "{fmt_extractor(extractor)}"', sub='extractors')  # noqa: E501
```

Pylint (3.3.7)

[convention] Line too long (156/100)

(C0301)